lib/middlewares/responseGenerator.js

```diff
@@ -142,6 +156,10 @@ function processAsynchronousRequest(req, res, next) {
             case 'GPS':
                 moduleAnswer = singleValueResponse(module);
                 break;
+            case 'L1':
```

Comment:
Just wanted to confirm with you that the same behaviour is expected for the `L1` and `AV` cases, since no `break` is included :)